### PR TITLE
Feature: Footer

### PR DESCRIPTION
**Pull request by moiskillnadne**

### CODE REVIEW

File: src/shared/Logo/ui/index.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { LOGO_URL } from "../lib/constants"
+
+export interface LogoProps {
+  widthPx?: number
+  heightPx?: number
```
Comment (BamMironov) [inline src/shared/Logo/ui/index.tsx:5]: Please define a separate interface `Dimension` with properties: `width` and `height` and use it here like this:
```
export interface LogoProps {
    size: Dimension;
    ...
}
```

File: src/shared/Logo/ui/index.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  classNameExtended?: string
```
Comment (BamMironov) [inline src/shared/Logo/ui/index.tsx:6]: No need to put the word "Extra" here. You can go with just `className` and it'll match with the React props style

File: src/shared/Logo/ui/index.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+const Logo = ({ widthPx, heightPx, classNameExtended }: LogoProps) => {
+  const defaultSize = {
```
Comment (BamMironov) [inline src/shared/Logo/ui/index.tsx:10]: Once you have defined the Dimension interface and used an appropriate prop you can default assign this object to that prop.
For example: 
```
const Logo = ({ size = { height: 50, width: 120, }, className }: LogoProps) {
...
```

File: src/shared/Logo/ui/index.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    width: 120,
+  }
+
+  const defaultClassName = "mr-1 p-1"
+
+  return (
+    <img
+      height={heightPx || defaultSize.height}
+      width={widthPx || defaultSize.width}
+      className={classNameExtended ? `${defaultClassName} ${classNameExtended}` : defaultClassName}
```
Comment (BamMironov) [inline src/shared/Logo/ui/index.tsx:21]: Please consider using [classnames](https://www.npmjs.com/package/classnames) or any decent alternative if you intend to make such class mixins.

**Review by BamMironov — CHANGES_REQUESTED** [review]

File: src/app/layout.tsx [hunk 5]
```diff
@@ -1,6 +1,7 @@
 import { ReactNode } from "react"
 import { useLocation } from "react-router-dom"
 
+import Footer from "../widgets/Footer"
 import Header from "../widgets/Header"
```
[the remark this answers is not in the document]
Comment (moiskillnadne) [reply src/app/layout.tsx:5]: Done

File: src/shared/Logo/ui/index.tsx (hunk 1 above)
Comment (moiskillnadne) [reply src/shared/Logo/ui/index.tsx:5]: Done

File: src/shared/Logo/ui/index.tsx (hunk 2 above)
Comment (moiskillnadne) [reply src/shared/Logo/ui/index.tsx:6]: Done

File: src/shared/Logo/ui/index.tsx (hunk 3 above)
Comment (moiskillnadne) [reply src/shared/Logo/ui/index.tsx:10]: It's great! I really forgot about it. Thanks, done.

File: src/shared/Logo/ui/index.tsx (hunk 4 above)
Comment (moiskillnadne) [reply src/shared/Logo/ui/index.tsx:21]: Done

File: src/shared/Logo/ui/index.tsx [hunk 6]
```diff
@@ -1,24 +1,21 @@
+import classnames from "classnames"
+
 import { LOGO_URL } from "../lib/constants"
+import { Dimension } from "../../../utils/types/dimension"
```
Comment (BamMironov) [inline src/shared/Logo/ui/index.tsx:4]: Please use absolute import

Comment (moiskillnadne) [reply src/shared/Logo/ui/index.tsx]: Done

**Review by BamMironov — APPROVED** [review]

**Pull request merged**